recipes/iiboost/build.sh

```diff
@@ -0,0 +1,19 @@
+
+#!/bin/bash
+
+if [[ "$(uname)" == "Darwin" ]]; then
+    export ENABLE_OPENMP=no
+else
+    export ENABLE_OPENMP=yes
+fi
+
+mkdir build && cd build
+cmake \
+    -DCMAKE_PREFIX_PATH=$PREFIX \
+    -DITK_DIR=$PREFIX/lib/cmake/ITK-4.11/ \
+    -DIIBOOST_USE_OPENMP=$ENABLE_OPENMP \
+	..
+
+make
```

Comment:
It also segfaults on osx, so I disable it there as well.

https://travis-ci.org/conda-forge/staged-recipes/builds/229909874